rs3-chunkpicker-chunkinfo-export.json

```diff
@@ -111439,7 +111466,7 @@
 			},
 			"Warforge Crucible": {
 				"Tasks": {
-					"Gain access to the ~|Warforge Dig Site|~": "Archaeology"
+					"Become an ~|archaeology assistant|~": "Archaeology"
```

Comment:
(Discussion) Shouldn't the req to enter the Warforge still be kept at "Gain access to the Warforge Dig Site", but have that one be changed to require being an assistant instead of the having the tutorial completed?
In fairness, I don't think there's a functional difference between the two.

Reply:
I've reverted the change... But I'm drawing a blank on how implement your suggestion tbh